src/seedu/addressbook/AddressBook.java

```diff
@@ -200,7 +200,18 @@
      */
     public static void main(String[] args) {
         showWelcomeMessage();
-        processProgramArgs(args);
```

Comment:
think about other constructs such as variable names that also contribute towards abstraction.